templates/threatconnect.hbs

```diff
@@ -967,23 +966,69 @@
                         }}
                         value={{orgData.indicator.__newCase.name}}
                         required
-                        disabled={{orgData.indicator.__isCreatingCase}}
+                        disabled={{or
+                          orgData.indicator.__isCreatingCase
+                          orgData.indicator.__newCase.__showDescriptionModal
+                        }}
                         {{on "input" (action "updateNewCaseField" indicatorId "name")}}
                       />
                       {{#if orgData.indicator.__newCase.__error}}
                         <div class="ifta-error-message">{{orgData.indicator.__newCase.__errorMessage}}</div>
                       {{/if}}
                     </div>
 
+                    <div class="form-group">
+                      <label for="case-workflowTemplate">Workflow Template</label>
```

Comment:
I would add a loading indicator here so the user knows that workflows have to load.  You probably want to disable the dropdown until the workflows are loaded.  You'll need to add an `isLoadingWorkflows` flag and then put a spinner here.
Something like:
```
<label for="case-workflowTemplate">
  Workflow Template
  <span class="p-footnote">
    {{fa-icon icon="spinner-third" fixedWidth=true spin=true}} Loading workflows ...
  </span>
</label>
```